### PR TITLE
Add Order Processed Date to csv

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/6546

[zgora-collective-transactions(6).csv](https://github.com/opencollective/opencollective-rest/files/10974641/zgora-collective-transactions.6.csv)

### CODE REVIEW

File: src/server/controllers/account-transactions.js [hunk 1]
```diff
@@ -261,6 +270,7 @@ const csvMapping = {
   payoutMethodType: (t) => get(t, 'expense.payoutMethod.type'),
   merchantId: (t) => get(t, 'merchantId'),
   orderMemo: (t) => get(t, 'order.memo'),
+  orderProcessedAt: (t) => formatOrderProcessedDate(t.order),
```
Comment (znarf) [inline src/server/controllers/account-transactions.js:273]:
```suggestion
  orderProcessedDate: (t) => formatOrderProcessedDate(t.order),
```

Comment (znarf) [reply src/server/controllers/account-transactions.js:273]: We should not need an utility for this.

Comment (znarf) [reply src/server/controllers/account-transactions.js:273]: Just do something like the `date` field.

Comment (SudharakaP) [reply src/server/controllers/account-transactions.js:273]: I've renamed the variable and removed the utility function.

Comment (SudharakaP) [reply src/server/controllers/account-transactions.js]: @znarf : Sorry, somehow missed commit due to error. I've just pushed the changes.

**Pull request merged**